File: backend/src/main/java/com/eskisehir/eventapi/domain/model/MobilityPreference.java
package com.eskisehir.eventapi.domain.model;

/**
 * User's preferred mode of transportation.
 * Affects route optimization constraints and carbon footprint calculation.
 */
public enum MobilityPreference {
    WALKING("Yürüyüş"),
    PUBLIC_TRANSPORT("Toplu Taşıma"),
    CAR("Araba"),
    BIKE("Bisiklet");

    private final String displayNameTr;

    MobilityPreference(String displayNameTr) {
        this.displayNameTr = displayNameTr;
    }

    public String getDisplayNameTr() {
        return displayNameTr;
    }
}

File: mobile/app/src/main/java/com/eskisehir/eventapp/data/model/MobilityPreference.kt
package com.eskisehir.eventapp.data.model

/**
 * User's preferred mode of transportation.
 * Affects route optimization constraints and carbon footprint calculation.
 */
enum class MobilityPreference(val displayNameTr: String) {
    WALKING("Yürüyüş"),
    PUBLIC_TRANSPORT("Toplu Taşıma"),
    CAR("Araba"),
    BIKE("Bisiklet")
}
